torchbenchmark/models/doctr_det_predictor/install.py

```diff
@@ -3,9 +3,6 @@
 import sys
 
 def pip_install_requirements():
-    # install deps from conda-forge
-    # model doctr_det_predictor needs weasyprint, which needs libglib and pango
-    subprocess.check_call(["conda", "install", "-y", "expecttest", "libglib", "pango", "-c", "conda-forge"])
```

Comment:
Could we just try and warn here instead of reverting the change entirely? maybe a
```
try:
    subprocess.check_call(["conda", "install", "-y", "expecttest", "libglib", "pango", "-c", "conda-forge"])
except Error:
    print("Failed to install weasyprint dependencies. Model doctr_det_predictor will not be able to be run.")
```

Reply:
I am thinking maybe we don't do this. This will make conda libraries override the libraries that already installed in user's environment.